framework/src/base/FEProblem.C

```diff
@@ -3582,11 +3573,10 @@ FEProblem::adaptMesh()
   unsigned int cycles_per_step = _adaptivity.getCyclesPerStep();
   for (unsigned int i=0; i < cycles_per_step; ++i)
   {
-    _console << "Adaptivity step " << i+1 << " of " << cycles_per_step << "\n";
+    _console << "Adaptivity step " << i+1 << " of " << cycles_per_step << std::endl;
```

Comment:
I disagree, that's exactly what a buffer is for, buffering output. Even though this branch of code is rarely called and it's not a point of slowness I think we should clean it up for consistency.

We should strive to cleanup the framework so that we flush when we need to see output and use normal newline characters where we don't. As long as meshChanged is printing to the console there is not need to flush here. If anything the flush should come "after" the mesh information is printed not after the header and before the body. I have removed dozens of unnecessary std::endls throughout the code, let's keep that going.